YELLOW_BELT/week_1/matrix_summ.cpp

```diff
@@ -1,107 +1,41 @@
-#include <fstream>
 #include <iostream>
-#include <stdexcept>
-#include <vector>
 
 class Matrix {
  public:
   int num_rows_;
   int num_columns_;
   int** arr;
-  std::vector<std::vector<int> > elements_;
 
   Matrix() {
     num_rows_ = 3;
     num_columns_ = 5;
     arr = new int*[num_rows_];
-    for (int i = 0; i < num_rows_; i++) {
+    for (int i = 0; i < num_rows_; ++i) {
       arr[i] = new int[num_columns_];
     }
   }
-
-  Matrix(int num_rows, int num_columns) {}
-
-  int At(int row, int column) const {
-    return elements_.at(row).at(column);
-  }
-
-  int& At(int row, int column) {
-    return elements_.at(row).at(column);
-  }
-
-  int GetNumRows() const {
-    return num_rows_;
-  }
-
-  int GetNumColumns() const {
-    return num_columns_;
+  ~Matrix() {
+    for (int i = 0; i < num_rows_; ++i) {
+      delete arr[i];
```

Comment:
@Ypostolov No right deleting.
Note! You are deleting two dimensional **array**

Reply:
Fixed in 9204b5c6dab6fc2f9ebd652d8302c06680e23c15.
Please check!